tiatoolbox/dataloader/wsireader.py

```diff
@@ -771,15 +772,21 @@ class OpenSlideWSIReader(WSIReader):
     """
 
     def __init__(
-        self, input_img=".",
+        self,
+        input_img=".",
```

Comment:
Default input for input_img?

Reply:
Updated in this commit https://github.com/TIA-Lab/tiatoolbox/commit/5276627cb0a863ab58b0e63001e9ad1b8dac5c3a